.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
**suggestion**: not needed for the first version, but you may want to automate the publishing using a github action. You may have a look at [how I configured it for the stylelint scss config](https://github.com/team23/stylelint-config-team23-scss/blob/master/.github/workflows/publish.yml).